polkadot/runtime/common/src/paras_registrar/mod.rs

```diff
@@ -613,7 +759,13 @@ impl<T: Config> Pallet<T> {
 		} else if let Some(rebate) = deposited.checked_sub(&deposit) {
 			<T as Config>::Currency::unreserve(&who, rebate);
 		};
-		let info = ParaInfo { manager: who.clone(), deposit, locked: None };
+		let info = ParaInfo {
+			manager: who.clone(),
+			deposit,
+			locked: None,
+			billing_account: Some(who.clone()),
```

Comment:
None?

Reply:
Maybe I am missing something, but setting this to `Some` immediately defeats the whole point of making this an `Option` - no?